HW_5/task_1.py

```diff
@@ -0,0 +1,15 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+
+# *Пример:*
+
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+def expon(a, b):
+    if (b == 1):
+        return (a)
+    if (b != 1):
+        return (a * expon(a, b - 1))
```

Comment:
Скобки тут не нужны.
```suggestion
    if b == 1:
        return a
    if b != 1:
        return a * expon(a, b - 1)
```